### PR TITLE
Small simplifications

### PR DESCRIPTION
**Pull request by GunnarFarneback**
A couple of small code simplifications.

### CODE REVIEW

**Review by ericphanson — APPROVED** [review]
Looks good to me!

bors merge

File: src/AutoMerge/jll.jl [hunk 1]
```diff
@@ -31,7 +31,7 @@ function meets_allowed_jll_nonrecursive_dependencies(working_directory::Abstract
                                                           pkg,
                                                           version)
     for dep in all_dependencies
-        if !((dep == "Pkg") | (dep == "Libdl") | (dep == "Artifacts") | (dep == "JLLWrappers") | (is_jll_name(dep)))
+        if dep ∉ ("Pkg", "Libdl", "Artifacts", "JLLWrappers") && !is_jll_name(dep))
```
Comment (ericphanson) [inline src/AutoMerge/jll.jl:34]:
```suggestion
        if dep ∉ ("Pkg", "Libdl", "Artifacts", "JLLWrappers") && !is_jll_name(dep)
```

**Comment by ericphanson** [comment]
bors merge

File: src/AutoMerge/jll.jl (hunk 1 above)
Comment (GunnarFarneback) [reply src/AutoMerge/jll.jl]: Oops.

**Pull request merged**